packages/wrangler/src/whoami.tsx

```diff
@@ -64,8 +64,8 @@ function Permissions(props: {
 					and re-login.
 				</Text>
 				<Text>Scope (Access)</Text>
-				{permissions.map(([type, name]) => (
-					<Fragment key={name}>
+				{permissions.map(([type, name], index) => (
+					<Fragment key={name + index}>
```

Comment:
Probably best to use `type + index` as `type` is always defined, whereas `name` can be undefined and `undefined + number` can lead us to some NaN issues

Reply:
Or `${type}${index}` would be fine too to avoid the addition